boards/arm/thingy91_nrf52840/Kconfig

```diff
@@ -1,14 +1,14 @@
-# Kconfig - nRF52840 PCA20035 board configuration
+# nRF52840 THINGY91 board configuration
```

Comment:
Perhaps `Thingy:91 nRF52840` instead. Same also other places and for the onboard nRF9160